test/specs/init.js

```diff
@@ -29,6 +29,21 @@ module.exports = function (promise) {
       expect(fsm.current).to.be.equal('green');
     });
 
+    it('should accept an Object for events property', function () {
+      StateMachine.Promise = promise;
+
+      var fsm = StateMachine({
+        events: {
```

Comment:
I don't *need* it this way, but it's less verbose if you don't have multiple events with the same name.  I didn't even realize that was supported.  So your call whether or not to support it--since it's trivial, why not?